src/main/java/th/ac/kmitl/science/comsci/example/models/CityName.java

```diff
@@ -0,0 +1,10 @@
+public class CityName {
+    String cityname;
```

Comment:
in Java naming best practice.

variables name must be writing in Lower Camel Case like `cityName`